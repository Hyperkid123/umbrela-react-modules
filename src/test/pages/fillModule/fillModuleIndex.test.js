import React from 'react'
import Landing from '../../../pages/fillModule/';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Fill module index component', () => {
  it('should render correctly', () => {
    const tree = shallow(
      <Landing/>
    )
    expect(toJson(tree)).toMatchSnapshot();
  })
});
