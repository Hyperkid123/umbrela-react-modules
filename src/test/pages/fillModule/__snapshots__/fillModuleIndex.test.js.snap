// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fill module index component should render correctly 1`] = `
<DragDropContext(DNDContext)>
  <Routes />
</DragDropContext(DNDContext)>
`;
